src/guides/libraries/c-interop.md

```diff
@@ -253,6 +253,65 @@ To reduce this burden,
 you can use the [`package:ffigen`][ffigen] binding generator
 to automatically create FFI wrappers from C header files.
 
+<a id="native-assets"></a>
+## Building and bundling native assets
+
+{{site.alert.note}}
+  The native assets feature is **experimental**,
+  and [in active development](https://github.com/dart-lang/sdk/issues/50565).
+{{site.alert.end}}
+
+The Native Assets feature aims to resolve a number of issues associated with
+the distribution of Dart packages that depend on native code.
+It does so by providing uniform hooks for integrating with various
+build systems involved in building Flutter and standalone Dart applications.
+
+The Native Assets feature aims to make it seamless for
+Dart packages to depend on and use native code:
+
+* It builds (if needed) the native code or
+  obtains the binaries using a package's `build.dart` script.
+* It bundles the native [`Asset`][] reported by the `build.dart` script.
+* It makes the native assets available at runtime through
+  declarative `@Native<>() extern` functions using the [`assetId`][].
+
+The `flutter run` / `flutter build` and `dart` / `dart build` tools
```

Comment:
Should this specify `dart run`? Or does only the base command support it currently?

```suggestion
The `flutter run` / `flutter build` and `dart run` / `dart build` tools
```

Reply:
Dart run is better.